libpod/container.go

```diff
@@ -588,3 +589,30 @@ func (c *Container) Export(path string) error {
 func (c *Container) Commit() (*storage.Image, error) {
 	return nil, ErrNotImplemented
 }
+
+// Wait blocks on a container to exit and returns its exit code
+func (c *Container) Wait() (int32, error) {
+	err := wait.PollImmediateInfinite(1,
+		func() (bool, error) {
+			if !c.isStopped() {
+				return false, nil
+			} else { // nolint
+				return true, nil // nolint
+			} // nolint
+		},
+	)
+
+	if err != nil {
+		return 0, err
+	}
+	exitCode := c.state.ExitCode
+	return exitCode, nil
+}
+
+func (c *Container) isStopped() bool {
```

Comment:
I still think this needs to return bool, err - we can't discard the error response from syncContainer(). It usually means the container was deleted elsewhere, so we'd keep waiting forever

Reply:
fixed